sections/sections-app-util/src/bundle/sections.properties

```diff
@@ -77,6 +79,7 @@ add_section_add_meeting=Add day(s) with a different meeting time and/or location
 section_table_category_header={0} Sections
 overview_table_header_remove=Remove
 options_confirm=The automatic setting will override any sections you have manually defined along with their memberships.  You will need to go to the Site Info tool and "Edit Rosters" to assign sections to the site for automatic membership management.  Would you like to continue?
+options_confirmInternal=Manually manage will prevent automatic updates from registration system. Do you want to continue?
```

Comment:
suggest better wording here:
This change will prevent automatic enrollment updates, are you sure you wish to continue?

Reply:
Updated